### ISSUE TITLE
Feature Request - Display last 10 submitted hashes

### ISSUE THREAD

**Issue by DeonBonthuys** [fireice-uk/xmr-stak#375]
Hi dev,

Love your miner way more than all the others!
I do ASIC mining as well and a nice feature is being able to see the actual number of accepted hashes like cgminer has.

In the console maybe show the actual number on every accepted result and on the html reports, have a section for the last 10 just like the top 10.

**Comment by fireice-uk** [comment]
It is already there. "pool-side hashes" are the hashes as seen by the pool.

**Comment by DeonBonthuys** [comment]
Yes, but, that is cumulative. Not a very friendly way to see each accepted hash. 


Sent from my T-Mobile 4G LTE Device
-------- Original message --------From: fireice-uk <notifications@github.com> Date: 12/10/17  5:52 PM  (GMT-05:00) To: fireice-uk/xmr-stak <xmr-stak@noreply.github.com> Cc: Deon Bonthuys <gfbonthuys@gmail.com>, Author <author@noreply.github.com> Subject: Re: [fireice-uk/xmr-stak] Feature Request - Display last 10 submitted
  hashes (#375) 
It is already there. "pool-side hashes" are the hashes as seen by the pool.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or mute the thread.


  
  




{"api_version":"1.0","publisher":{"api_key":"05dde50f1d1a384dd78767c55493e4bb","name":"GitHub"},"entity":{"external_key":"github/fireice-uk/xmr-stak","title":"fireice-uk/xmr-stak","subtitle":"GitHub repository","main_image_url":"https://cloud.githubusercontent.com/assets/143418/17495839/a5054eac-5d88-11e6-95fc-7290892c7bb5.png","avatar_image_url":"https://cloud.githubusercontent.com/assets/143418/15842166/7c72db34-2c0b-11e6-9aed-b52498112777.png","action":{"name":"Open in GitHub","url":"https://github.com/fireice-uk/xmr-stak"}},"updates":{"snippets":[{"icon":"PERSON","message":"@fireice-uk in #375: It is already there. \"pool-side hashes\" are the hashes as seen by the pool. "}],"action":{"name":"View Issue","url":"https://github.com/fireice-uk/xmr-stak/issues/375#issuecomment-350588417"}}}

**Comment by fireice-uk** [comment]
You aren't loosing out. If you double the difficulty the shares will come 2x slower. I will log it on the todo
